ui/src/views/Admin.vue

```diff
@@ -129,6 +135,15 @@ export default {
         this.dataTXT = await axios.get(`${getAPIRoot()}/sample-config?type=cfg`).then((res) => res.data)
       } else if (this.flag === 'template_aic') {
         this.dataTXT = await axios.get(`${getAPIRoot()}/sample-config?type=aic`).then((res) => res.data)
+      } else if (this.flag === 'Remove_lock_file') {
+        const res = await api.runCommand('script_remove_lock_file')
+        if (res === true) {
+          this.$notify({
+            type: 'success',
+            title: 'Success',
+            text: 'Lock file has been deleted successfully!',
```

Comment:
We discussed to rename the message to "Action triggered successfully", as you have to know that SVTrain only triggers scripts that are subject to whatever logic.

Reply:
@fk-230119-sc-kpi, let me quickly fix it.